src/lang/modifyAst.test.ts

```diff
@@ -601,15 +601,15 @@ ${!replace1 ? `  |> ${line}\n` : ''}  |> angledLine(angle = -65, length = ${
       !replace1 ? 'segLen(a)' : replace1
     })
   |> line(end = [306.21, 198.87])
-  |> angledLine(angle = 65, length = ${!replace2 ? 'segAng(a)' : replace2})
+  |> angledLine(angle = ${!replace2 ? 'segAng(a)' : replace2}, length = 300)
```

Comment:
Interesting. This was a KCL runtime error before this change?

Reply:
There was no error as such we were using an angle as a length, but that was fine, now an error.

Reply:
Sorry, that's what I meant.